frontend/lib/src/components/widgets/TextInput/TextInput.tsx

```diff
@@ -105,7 +105,7 @@ class TextInput extends React.PureComponent<Props, State> {
    * Commits the current state value to the WidgetStateManager.
    *
    * @param source - Whether or not from the UI
-   * @param updateDirtyState - Optional flag to determine if the state should be updated
+   * @param updateState - Optional flag to determine if the state should be updated
    *                           to reflect that the value is no longer 'dirty' or modified.
    *                           By default, this is true, meaning the state WILL be updated.
```

Comment:
The docstring indentation here is misaligned

Reply:
Oops thanks. Will correct that.